src/content/recipes/tailwindcss.md

```diff
@@ -139,67 +132,28 @@ module.exports = {
 };
 ```
 
-To add the switcher, declare a [global type](/docs/react/essentials/toolbars-and-globals) named "theme" in `.storybook/preview.js` and give it a list of supported themes to choose from.
+To add the switcher, add the [`withThemeFromDataAttribute`](https://github.com/storybookjs/addon-styling/blob/main/docs/api.md#withthemebydataattribute) decorator to your storybook from `@storybook/addon-styling`
 
 ```js
 // .storybook/preview.js
-export const globalTypes = {
-  theme: {
-    name: 'Theme',
-    description: 'Global theme for components',
-    toolbar: {
-      icon: 'paintbrush',
-      // Array of plain string values or MenuItem shape
-      items: [
-        { value: 'light', title: 'Light', left: '🌞' },
-        { value: 'dark', title: 'Dark', left: '🌛' },
-      ],
-      // Change title based on selected value
-      dynamicTitle: true,
-    },
-  },
-};
-```
-
-This code will create a new toolbar menu to select your desired theme for your stories.
-
-### Add a `withTailwindTheme` decorator
-
-There needs to be a way to tell Tailwind to use the theme that is selected in the toolbar. To do that, This can be done using a [decorator](/docs/vue/writing-stories/decorators).
-
-Below I created a new file in `.storybook` called `withTailwindTheme.decorator.js` that will take the global theme value and update the current theme.
-
-```js
-// .storybook/withTailwindTheme.decorator.js
-
-import { useEffect } from 'react';
-
-export const DEFAULT_THEME = 'light';
-
-export const withTailwindTheme = (Story, context) => {
-  const { theme } = context.globals;
-
-  useEffect(() => {
-    const htmlTag = document.documentElement;
-
-    // Set the "data-mode" attribute on the iFrame html tag
-    htmlTag.setAttribute('data-mode', theme || DEFAULT_THEME);
-  }, [theme]);
-
-  return <Story />;
-};
-```
-
-Now all we have to do is give this decorator to Storybook to wrap our stories in. Add the decorator to the `decorators` array in `.storybook/preview.js`:
-
-```js
-import { DEFAULT_THEME, withTailwindTheme } from './withTailwindTheme.decorator';
+import { withThemeByDataAttribute } from '@storybook/addon-styling';
 
 /* snipped for brevity */
 
-export const decorators = [withTailwindTheme];
+export const decorators = [
+  withThemeFromDataAttribute({
+    themes: {
+      light: 'light',
+      dark: 'dark',
+    },
+    defaultTheme: 'light',
+    attributeName: 'data-mode',
+  }),
+];
 ```
 
+This code will create a new toolbar menu to select your desired theme for your stories.
+
 ## Get involved
 
 Now you're ready to use Tailwind with Storybook. 🎉 Check out the [example repo](https://github.com/Integrayshaun/storybook-tailwind-recipe-example) for a quick start.
```

Comment:
_I can't comment on `bootstrap.svg`, so I'm doing so here._

Wrong PR for this file to be added?